src/material/core/focus_indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,54 @@
+@import '../theming/theming';
+
+@mixin mat-focus-indicator() {
```

Comment:
I'll start the obligatory name bike-shedding thread. Here's a couple suggestions:
- `mat-accessible-focus-indicators()`
- `mat-high-contrast-focus-indicators()`

I like `indicators` plural because if its not plural it sounds like a mixin that's used to add focus indicator styles to an element, not a mixin used to apply styles to multiple components across the board.

I'm also in favor of something in the name (e.g. `accessible`, `high-contrast`) that conveys the fact that this mixin is not for applying the standard focus treatment, but something more beyond that.

Reply:
I was thinking something like `mat-strong-focus-indicators()`. I wouldn't want to use `high-contrast` since that's overloading with Windows High Contrast mode. "Accessible" doesn't feel right because it implies the default treatment isn't accessible _at all_, when it's more of a spectrum.

Reply:
Is there a better place for these mixins to live?